pkg/util/validation/validation.go

```diff
@@ -125,41 +125,29 @@ func ValidateSpreadConstraint(spreadConstraints []policyv1alpha1.SpreadConstrain
 }
 
 // ValidateOverrideSpec validates that the overrider specification is correctly defined.
-func ValidateOverrideSpec(overrideSpec *policyv1alpha1.OverrideSpec) error {
+func ValidateOverrideSpec(overrideSpec *policyv1alpha1.OverrideSpec) field.ErrorList {
+	var allErrs field.ErrorList
 	if overrideSpec == nil {
 		return nil
 	}
-
-	if overrideSpec.OverrideRules == nil {
-		return nil
+	specPath := field.NewPath("spec")
+	//nolint:staticcheck
+	// disable `deprecation` check for backward compatibility.
```

Comment:
why disable lint check here?

Reply:
Because the `TargetCluster`  field is deprecated in v1.0.